Makefile

```diff
@@ -11,3 +13,7 @@ lintf:
 build:
 	prototool all
 	go mod tidy
+	for dir in $(SRC_DIR); \
+	do \
+	eval go build $$dir; \
```

Comment:
does it work with just `go build $(dir)`?

Reply:
waiting for a reason on eval, normally it's unsafe to use

Reply:
yes, without eval it also works, let's me remove this eval